activejob/lib/active_job/arguments.rb

```diff
@@ -72,10 +72,10 @@ def serialize_argument(argument)
           convert_to_global_id_hash(argument)
         when Array
           argument.map { |arg| serialize_argument(arg) }
+        when ActionController::Parameters
```

Comment:
I think this line here is what's causing the tests to fail. I imagine it's because `activejob` doesn't have a dependency on `actioncontroller`, which makes a lot of sense when you think about it.

Reply:
hmmm...interesting, maybe I need to check the `ActionController` is included first in `ActiveJob`